api/peripheral_api/routes/Cleezy.js

```diff
@@ -33,13 +33,16 @@ router.get('/listAll', async (req, res) => {
     return res.sendStatus(UNAUTHORIZED);
   }
   try {
-    const response = await axios.get(CLEEZY_URL + '/list');
+    const page = req.query.page || 0;
```

Comment:
if you wanna be cool
```suggestion
    const { page = 0 } = req.query;
```

Reply:
done